### PR TITLE
Refactor cors

### PR DESCRIPTION
**Pull request by freddik**
develop

## GitHub Board

**Issue link**

[#11 Issue name ]( )

**Story link**

[#11 Story]( )

## Code reviewers
- [ ] @github_username

## Summary of issue

- [ ]  Summary of issue
- [ ]  Summary of issue

## Summary of change

- [ ]  Summary of change
- [ ]  Summary of change

### CODE REVIEW

File: src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/**")
-                .allowedOrigins("*")
+                .allowedOrigins("http://localhost:3000", "https://dokazovi-fe.herokuapp.com")
```
[the remark this answers is not in the document]
Comment (freddik) [reply src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java:15]: yes, shure

**Review by ad-artem-melnykov — APPROVED** [review]

File: src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java [hunk 2]
```diff
@@ -9,10 +10,13 @@
 
     private static final long MAX_AGE_SEC = 3600;
 
+    @Value("${endpoints.cors.allowed-origins}")
+    private String [] allowedOrigins;
```
Comment (ad-artem-melnykov) [inline src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java:14]: Did you test it? Your build is failed

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -3,6 +3,7 @@ logging.level.io.swagger.models.parameters.AbstractSerializableParameter=ERROR
 spring.jackson.serialization.fail-on-empty-beans=false
 server.servlet.context-path=/api
 server.port=${PORT:8080}
+endpoints.cors.allowed-origins=http://localhost:3000, https://dokazovi-fe.herokuapp.com
```
Comment (ad-artem-melnykov) [inline src/main/resources/application.properties:6]: Please try to use it with the injection variable:
endpoints.cors={ALLOWED_ORIGIN: http://localhost:3000, https://dokazovi-fe.herokuapp.com}

File: src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java (hunk 2 above)
Comment (freddik) [reply src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java:14]: Yes, everything was ok

Comment (ad-artem-melnykov) [reply src/main/java/com/softserveinc/dokazovi/config/CorsConfig.java:14]: https://github.com/ita-social-projects/dokazovi-be/pull/191/checks?check_run_id=2023305740

File: src/main/resources/application.properties (hunk 3 above)
Comment (freddik) [reply src/main/resources/application.properties:6]: I have the same error as you do in this case

Comment (freddik) [reply src/main/resources/application.properties:6]: My version works

Comment (andrewtyshkovets) [reply src/main/resources/application.properties:6]: Also we were talking about array of strings, so please investigate if we need it

Comment (freddik) [reply src/main/resources/application.properties:6]: But I use this one - "private String [] allowedOrigins".  Did I misunderstand something?

Comment (andrewtyshkovets) [reply src/main/resources/application.properties:6]: I just don't see split function

**Review by melnikovae87 — APPROVED** [review]

**Review by andrewtyshkovets — APPROVED** [review]

**Pull request merged**